tests/tests.c

```diff
@@ -1431,10 +1436,43 @@ bool sirtest_os_log(void) {
 #endif
 }
 
+char *_sirtest_wine(void) {
+#if defined(__WIN__)
+# if defined(_MSC_VER) && !defined(__clang__)
+#  pragma warning(push)
+#  pragma warning(disable : 4152)
+# endif
+    static const char *(CDECL *_p_wine_get_version)(void);
+    HMODULE _h_ntdll;
+
+    _h_ntdll = GetModuleHandle("ntdll.dll");
+    if (_h_ntdll != NULL) {
+# if defined(__GNUC__)
+#  pragma GCC diagnostic push
+#  pragma GCC diagnostic ignored "-Wpedantic"
+# endif
+        _p_wine_get_version = (void *)GetProcAddress(_h_ntdll, "wine_get_version");
+# if defined(__GNUC__)
+#  pragma GCC diagnostic pop
+# endif
+        char *wine_version = (char *)_p_wine_get_version();
+        if (wine_version)
+            return wine_version;
+    }
+# if defined(_MSC_VER) && !defined(__clang__)
+#  pragma warning(pop)
+# endif
+#endif
+    return NULL;
+}
+
 bool sirtest_filesystem(void) {
     INIT(si, SIRL_ALL, 0, 0, 0);
     bool pass = si_init;
 
+    /* Wine version */
+    printf("\t_sirtest_wine: '%s'\n", PRN_STR(_sirtest_wine()));
```

Comment:
Maybe have this say Running under Wine: %s...", (NULL != winever) ? "yes" : "no"

Reply:
That didn't make sense. Maybe make it say "Wine version: <version> or <not under Wine>"

Reply:
We'll try `printf("\tRunning under Wine: %s\n", sirtest_get_wineversion() ? sirtest_get_wineversion() : "no");`

```
(1/1) 'filesystem'...                                            
                                
        Running under Wine: no
```

Or else it'll show Running under Wine: 8.11 ...